docs/DeveloperGuide.md

```diff
@@ -1441,24 +1441,54 @@ Deleting sessions in the students tab
 
 { end of `manual_testing#deleting_a_session` written by: Marcus Tan Wei }
 
-{ start of `appendix` written by: _________ }
+{ start of `appendix` written by: Masagca Eris Jacey }
 
 ## 12. Appendix: effort
 
 **Difficulty level:**
 
-* todo
+The overall difficulty level of the project was relatively moderate. 
```

Comment:
Should this be relatively difficult? :DDD

Reply:
Will be changing to that